### PR TITLE
docs: REST api check

### PR DESCRIPTION
**Pull request by kimberlyxdavis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

### CODE REVIEW

File: docs/General/buildRPC.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Read more about GitHub Actions [here](https://github.blog/2019-08-08-github-acti
 
 3. Click the **Run Workflow** dropdown. 
 
-4. Select the **branch: v1.2.2** from the **Use workflow from** dropdown. 
+4. Select the **branch: v1.2.2** from the **Use workflow from** dropdown. If you don't see it in the list, type "v1.2.2" to find the correct branch.
```
Comment (matt-primrose) [inline docs/General/buildRPC.md:34]: Currently we are at 1.2.1.  I'm wondering if we should rephrase this to avoid using specific version numbers and instead use latest release version as the branch value?

Comment (kimberlyxdavis) [reply docs/General/buildRPC.md:34]: Good idea.

**Review by matt-primrose — APPROVED** [review]

**Review by bwendlandt-intel — APPROVED** [review]
Looks good.  You'll prob have to resolve a conflict with the .yml.  Go with whatever naming convention you think fits best between the two since you were a lot closer to those scale docs.
